src/store/getters.ts

```diff
@@ -102,13 +102,37 @@ export default {
     state.gridVariables === null ? null : state.gridVariables.map((variable: VariableWithVariants) =>
       getters.gridAssessments.map((assessment: Assessment) => {
         if (state.variantCounts === null) { return NaN }
+
         const variants: Variant[] = variable.variants.filter((variant: Variant) => variant.assessmentId === assessment.id)
-        const count: number = variants.reduce((sum: number, variant: Variant) => {
-          // @ts-ignore
-          const variantCount = state.variantCounts.find((variantCount) => variant.id === variantCount.variantId)
-          return sum + (variantCount ? variantCount.count : 0)
-        }, 0)
-        return count
+
```

Comment:
the reduce seems more clear to me is this case , as we are reducing a variant list to a single number

Reply:
when i revert this change in de getters.ts the unit tests still all pass and the reported issue in fixed ..

Reply:
there were some usecases not tested, now fixed